package.json

```diff
@@ -27,7 +27,7 @@
     "@types/express": "^4.0.33",
     "@types/mime": "0.0.29",
     "@types/mz": "0.0.29",
-    "@types/node": "^4.0.30",
+    "@types/node": "=4.0.30",
```

Comment:
Isn't it true that the `=` is not a valid option in the depedencies? To have an exact version, there should only be a version. I couldn't find it in the [docs](https://docs.npmjs.com/files/package.json#dependencies).